Makefile

```diff
@@ -61,6 +61,10 @@ endif
 LOCAL_ARCH := $(shell uname -m)
 ifeq ($(LOCAL_ARCH),x86_64)
 GOARCH_LOCAL := amd64
+else ifeq ($(shell echo $(LOCAL_ARCH) | head -c 5),armv8)
+GOARCH_LOCAL := arm64
+else ifeq ($(shell echo $(LOCAL_ARCH) | head -c 4),armv)
+GOARCH_LOCAL := arm
```

Comment:
is this arm32?  If so, I think this can execute a warning indicating 32 bit Istio is not available nor supported.

Reply:
yes `arm32`, I can build it properly without seeing any warning